csp-mixin/panels/azurefrontdoor.libsonnet

```diff
@@ -0,0 +1,300 @@
+local g = import '../g.libsonnet';
+local commonlib = import 'common-lib/common/main.libsonnet';
+{
+  new(this): {
+    // Azure Front Door
+
+    afd_endpoint_count:
+      this.signals.azurefrontdoorOverview.endpointCount.asStat()
+      + commonlib.panels.generic.stat.base.stylize(),
+
+    afd_top5_errors:
+      this.signals.azurefrontdoorOverview.top5Errors.asTable(format='table')
+      + commonlib.panels.generic.table.base.stylize()
+      + g.panel.table.standardOptions.withOverrides(
+        [
+          {
+            matcher: {
+              id: 'byName',
+              options: 'Value',
+            },
+            properties: [
+              {
+                id: 'custom.width',
+                value: 80,
+              },
+              {
+                id: 'color',
+              },
+              {
+                id: 'custom.cellOptions',
+                value: {
+                  type: 'gauge',
+                  valueDisplayMode: 'color',
+                },
+              },
+              {
+                id: 'unit',
+                value: 'percent',
+              },
+              {
+                id: 'thresholds',
+                value: {
+                  mode: 'absolute',
+                  steps: [
+                    {
+                      color: 'yellow',
+                      value: null,
+                    },
+                    {
+                      color: 'green',
+                      value: 30,
+                    },
+                    {
+                      color: 'red',
+                      value: 85,
+                    },
```

Comment:
should we switch `yellow` with `green` thresholds? so that when there are 0 errors the cell is green, then above ~~30~~ 15 it turns yellow and above (perhaps update the threshold to 50) it turns red. what do you think?

Reply:
I think if it's null, it can be text-color. Then anything above it can be red